extension/src/extension.ts

```diff
@@ -165,19 +165,18 @@ async function getVscodeTerminal(name = ""): Promise<vscode.Terminal | null> {
 
 async function runCodeNatively(document: vscode.TextDocument) {
   const currentTerminalType = getTerminalType();
-  const isGitBash = currentTerminalType === "Git Bash";
   if (
     !(
       currentTerminalType === "Command Prompt" ||
-      currentTerminalType === "PowerShell" ||
-      isGitBash
+      currentTerminalType === "PowerShell"
     )
   ) {
     vscode.window.showInformationMessage(
       "Your current default terminal is unsupported please change your default terminal to use MASM Runner extension"
     );
     return;
   }
+  const isCMD = currentTerminalType === "Command Prompt";
```

Comment:
why not just declare this inbetween line168/169 and use it in the check?

Reply:
i think that would've been better I didn't think of that but that is an obvious solution as well